### PR TITLE
handle choral view errors more gracefully

### PR DESCRIPTION
**Pull request by jboretsky**

### CODE REVIEW

**Comment by jboretsky** [comment]
fixes #116

File: routes/chorals.js [hunk 1]
```diff
@@ -250,8 +285,8 @@ router.get('/:choralId', function(req, res, next) {
       cass_client.execute( query , [ choralId,  1 + (timeFrame/parentObj.choralInfo.sampleRate)], { prepare: true }, function( err, result ) {
         if(err || !result) {
           logHelper.createLog("error", 'Choral data was not found in cassandra: ' + err, ["chorals", "get"]);
-          console.log(err);
-          res.send("Choral not found in database");
+          parentObj.errorMsg = "Choral was not found in database, how did you get here?";
```
Comment (james-woo) [inline routes/chorals.js:275]: I like this :yum:

File: routes/chorals.js [hunk 2]
```diff
@@ -141,8 +141,6 @@ router.get('/:choralId', function(req, res, next) {
     Choral.findOne({ choralId: choralId }, (err, choral) => {
       if (err || !choral) {
         logHelper.createLog("error", 'Choral does not exist: ' + err, ["chorals", "get"]);
```
Comment (james-woo) [inline routes/chorals.js:145]: If the choral doesn't exist, do we only want to show 404?

Comment (jboretsky) [reply routes/chorals.js:145]: I think it's reasonable to just show a 404 if someone types .../chorals/1234 but I guess we could just show an error  choral page. The thing is if we don't even have it in mongo, i have 0 information to populate the page with

**Comment by cpjk** [comment]
:eyes:

File: routes/chorals.js [hunk 3]
```diff
@@ -182,13 +189,21 @@ router.get('/:choralId', function(req, res, next) {
...
+          if(err || data == null) {
             logHelper.createLog(
               "error",
               'Choral data has not yet been published to redis: ' + err
               , ["chorals", "get"]
             );
-            reject(err);
+            dataObj = {
+              choralInfo: {
+                sampleRate: choral.sampleRate,
```
Comment (cpjk) [inline routes/chorals.js]: I'm getting `choral is not defined` here. It looks like  `choral` is never available in this scope AFAICT.

Comment (jboretsky) [reply routes/chorals.js]: right mb it isn't defined here

**Comment by cpjk** [comment]
i think this looks good.

**Pull request merged**